src/Program.cs

```diff
@@ -1,28 +1,46 @@
 ﻿using ApiSdk;
 using Azure.Identity;
+using Microsoft.Graph.Cli.Authentication;
+using Microsoft.Graph.Cli.Commands.Authentication;
 using Microsoft.Graph.Cli.Utils;
 using Microsoft.Kiota.Authentication.Azure;
 using Microsoft.Kiota.Http.HttpClientLibrary;
 using System.CommandLine;
+using System.Net.Http;
+using System.Reflection;
 using System.Threading.Tasks;
 
 namespace Microsoft.Graph.Cli
 {
     class Program {
+        const string SdkVersionHeaderValueFormatString = "{0}-cli-{1}.{2}.{3}";
```

Comment:
`{0}-cli-{1}.{2}.{3}` becomes `graph-cli/{1}.{2}.{3}`

This requires change on L28

Reply:
Yes, thank you!